emoji/src/main/java/chat/rocket/android/emoji/internal/db/EmojiDatabase.kt

```diff
@@ -0,0 +1,48 @@
+package chat.rocket.android.emoji.internal.db
+
+import android.content.Context
+import androidx.room.Database
+import androidx.room.Room
+import androidx.room.RoomDatabase
+import androidx.room.TypeConverters
+import chat.rocket.android.emoji.Emoji
+import chat.rocket.android.emoji.EmojiDao
+
+@Database(entities = [Emoji::class], version = 1)
+@TypeConverters(StringListConverter::class)
+abstract class EmojiDatabase : RoomDatabase() {
```

Comment:
Just pushed an update. Instead of creating a database per server I changed the query for custom emojis taking account the current server url (a custom emoji is always stored on the current server).
When one uploads a custom emoji image to the server it goes to the server's instance. If by any chance, this changes in the future (for instance, uploaded emojis being stored on aws), then we should create a new db or maybe create a currentServer property on the entity prior to saving it.
I think it looks good for now, I just have to adjust the recents to also contemplate this change (it's still being saved on sharedpreferences). This logic never changed, old bug (it didn't matter much before because we didn't have custom emojis).

Reply:
Done with recents.